nextest-runner/src/runner/dispatcher.rs

```diff
@@ -919,7 +937,9 @@ mod tests {
             vec![],
             0,
             MaxFail::All,
+            Instant::now() + Duration::from_secs(86400 * 365 * 30),
```

Comment:
Could this reuse `far_future` from `pausable_sleep.rs`?

Reply:
what about adding a far_future_duration function?

Reply:
Done.